cmd/atlas/internal/cmdlog/cmdlog.go

```diff
@@ -785,6 +786,15 @@ func (s *SchemaInspect) MarshalHCL() (string, error) {
 	return string(spec), nil
 }
 
+// RedactedURL returns the inspected url redacted.
+func (s *SchemaInspect) RedactedURL() (string, error) {
```

Comment:
It is dangerous to have the URL still be un-redacted. I advise to have it `URL`, unexport the `url` and add it as either arg or optional arg to the constructor.